generator/views/metric_definitions_view.py

```diff
@@ -253,7 +255,7 @@ def to_lookml(self, bq_client, v1_name: Optional[str]) -> Dict[str, Any]:
             """
         }
 
-        view_defn["dimensions"] = self.get_dimensions()
+        view_defn["dimensions"] = self.get_dimensions(dryrun=dryrun)
```

Comment:
I don't think `dryrun` should be passed here since this `get_dimensions()` implementation doesn't use it.
```suggestion
        view_defn["dimensions"] = self.get_dimensions()
```